x-pack/plugins/reporting/public/components/report_info_button.tsx

```diff
@@ -86,102 +87,95 @@ export class ReportInfoButton extends Component<Props, State> {
     const maxAttempts = info.max_attempts ? info.max_attempts.toString() : NA;
     const priority = info.priority ? info.priority.toString() : NA;
     const timeout = info.timeout ? info.timeout.toString() : NA;
-    const warnings = info.output && info.output.warnings ? info.output.warnings.join(',') : null;
-
-    const jobInfoDateTimes: JobInfo[] = [
-      {
-        title: 'Created By',
-        description: info.created_by || NA,
-      },
-      {
-        title: 'Created At',
-        description: info.created_at || NA,
-      },
-      {
-        title: 'Started At',
-        description: info.started_at || NA,
-      },
-      {
-        title: 'Completed At',
-        description: info.completed_at || NA,
-      },
-      {
-        title: 'Processed By',
-        description:
-          info.kibana_name && info.kibana_id ? `${info.kibana_name} (${info.kibana_id})` : UNKNOWN,
-      },
-      {
-        title: 'Browser Timezone',
-        description: get(info, 'payload.browserTimezone') || NA,
-      },
-    ];
-    const jobInfoPayload: JobInfo[] = [
-      {
-        title: 'Title',
-        description: get(info, 'payload.title') || NA,
-      },
-      {
-        title: 'Type',
-        description: get(info, 'payload.type') || NA,
-      },
-      {
-        title: 'Layout',
-        description: get(info, 'meta.layout') || NA,
-      },
-      {
-        title: 'Dimensions',
-        description: getDimensions(info),
-      },
-      {
-        title: 'Job Type',
-        description: jobType,
-      },
-      {
-        title: 'Content Type',
-        description: get(info, 'output.content_type') || NA,
-      },
-      {
-        title: 'Size in Bytes',
-        description: get(info, 'output.size') || NA,
-      },
-    ];
-    const jobInfoStatus: JobInfo[] = [
-      {
-        title: 'Attempts',
-        description: attempts,
-      },
-      {
-        title: 'Max Attempts',
-        description: maxAttempts,
-      },
-      {
-        title: 'Priority',
-        description: priority,
-      },
-      {
-        title: 'Timeout',
-        description: timeout,
-      },
-      {
-        title: 'Status',
-        description: info.status || NA,
-      },
-      {
-        title: 'Browser Type',
-        description: USES_HEADLESS_JOB_TYPES.includes(jobType) ? info.browser_type || UNKNOWN : NA,
-      },
-    ];
-    if (warnings) {
-      jobInfoStatus.push({
```

Comment:
Oops, did the `Errors` get lost in a merge?

Reply:
Interesting... I'm trying to find where this went, I don't recall directly altering it so it's likely something happened.

Reply:
I think just all the `jobInfoParts` stuff needs to be reverted